yelp_package/extra_requirements_yelp.txt

```diff
@@ -1,4 +1,5 @@
 --extra-index-url=https://pypi.yelpcorp.com/simple
+clusterman_metrics==1.2.7
```

Comment:
this will need to bump to the new version of clusterman_metrics once I release it